butane_cli/src/lib.rs

```diff
@@ -196,13 +196,15 @@ pub fn embed(base_dir: &Path) -> Result<()> {
     let json = serde_json::to_string(&mem_ms)?;
```

Comment:
I would also like to change this to to_string_pretty, so that the generated `butane_migrations.rs` is more readable when there are changes to the migrations.